telemetry/derive/Cargo.toml

```diff
@@ -26,6 +26,6 @@ proc-macro2 = "1.0"
 proc-macro-error = "1.0"
 
 [dev-dependencies]
-iroha_core = { path = "../../core" }
+iroha_core = { version = "=2.0.0-pre-rc.2", path = "../../core" }
```

Comment:
managing versions manually like this seems problematic. Could we consider some automated way to bump versions across all crates? also it doesn't seem too important to specify versions for `dev-dependencies`?

Reply:
I didn't do it manually. The crates are modified in lockstep, and changing `=vX.X.X` to `=vX.X.X+1` is a simple text replacement.